.circleci/config.yml

```diff
@@ -25,7 +25,8 @@ jobs:
     steps:
       - attach_workspace:
           at: /tmp/workspace
-      - run: pip install -e/tmp/workspace/wagtail-localize[testing]
+      - run: pip install pip>=21.3
+      - run: pip install -e /tmp/workspace/wagtail-localize[testing]
```

Comment:
nitpick: this is not wagtail-localize

Reply:
It is not, but this code is right.
If you read the full file, this CI build starts by running `cookiecutter . --no-input`, which generates a project, accepting the default answer for every option. Then having generated a package, it runs some checks on the output (i.e: without the `{{ placeholders }}`).
The default for `project_name` is `Localize` so the package it spits out is called wagtail-localize (and it is authored by Karl Hobley, etc).
..or is your point that we should change the defaults?

Also, note to self: This Circle CI build appears to be running on pushes to `main` but not on PRs. The build failed on https://github.com/wagtail/cookiecutter-wagtail-package/commit/cc2d4eb00985747fdd524c8448b1776b817ba44f after we merged it but it didn't get picked up in review. I'll have a look at sorting this as well, unless it requires permissions I don't have in Circle CI. This is a problem for FutureChris to deal with..

Reply:
Sorry, I did this thing where I skipped the middle part 🤦‍♂️
The gist of it was that we should move away from using localize as the default and instead use something more generic.

As an aside (and subject to a different PR), I would like to move to GitHub Actions for the main tests. Circle CI is great, but it is one more moving part which makes sense for a large codebase (e.g. Wagtail) to run different tests in Circle CI and GitHub Actions, but for most packages we should keep the moving parts to a minimum

Reply:
Yeah agreed with both. I see you've already raised https://github.com/wagtail/cookiecutter-wagtail-package/issues/34
I've also raised https://github.com/wagtail/cookiecutter-wagtail-package/issues/35 to replace the defaults with something more.. defaulty.
I don't think we should deal with either of them in this PR